implementations/rust/ockam/ockam_api/src/nodes/models/portal.rs

```diff
@@ -23,15 +23,15 @@ pub struct CreateInlet<'a> {
     /// The peer address.
     /// This can either be the address of an already
     /// created outlet, or a forwarding mechanism via ockam cloud.
-    #[n(2)] outlet_addr: MultiAddr,
+    #[b(2)] outlet_addr: MultiAddr,
```

Comment:
!

Reply:
Done 👍 

Interestingly, there was also a `listen_addr: SocketAddr` in this struct (introduced by twittner) which was borrowing. Following our logic, I changed it to non-borrowing.

Reply:
Borrowing probably worked in this case because `SocketAddr` implements `Copy`.